carbonmark-api/src/models/Project.model.ts

```diff
@@ -21,6 +21,14 @@ export const ProjectModel = Type.Object({
   country: Type.Object({ id: Type.String() }),
   price: Type.String(),
   listings: Nullable(Type.Array(ListingModel)), // null when listings are empty
+  images: Nullable(
```

Comment:
The map view is using them for the cards however maybe this should be fetched dynamically to avoid blowing up the `/projects` payloads (on render of the map marker) 

<img width="417" alt="Screenshot 2023-09-15 at 5 58 49 am" src="https://github.com/KlimaDAO/klimadao/assets/7104689/6835b580-1368-4f33-90c0-4dd312edc297">

Thoughts?

Reply:
oh didn't realize the map had images, neat. Then we can keep them in the response & keep it simple for now. There aren't that many projects with images yet so its fine for now.